docs/articles/documentation/guides/continuous-integration/travis.md

```diff
@@ -38,52 +37,34 @@ Install TestCafe [locally](../basic-guides/install-testcafe.md#local-installatio
 
 3. Add a `.travis.yml` configuration file to the root of your project. This file contains parameters and commands that instruct Travis CI how to execute your builds. For more information, see [Customizing the Build](https://docs.travis-ci.com/user/customizing-the-build).
 
-     For Node.js projects, the `.travis.yml` file can have the following content.
+     For Node.js projects that require Firefox and Chrome installed, the `.travis.yml` file can have the following content:
 
      ```yaml
      language: node_js
      node_js: "stable"
   
+     dist: bionic
+
      before_install:
         - stty cols 80
+
+     addons:
+        firefox: latest
+        chrome: stable
      ```
 
      Commit and push this file to your repository.
 
-## Step 3 - Configure Travis to use Xvfb
-
-Travis CI uses Ubuntu Server virtual machines that do not have regular graphical environment like Unity, GNOME or KDE installed, so you have to setup and use [Xvfb](https://www.x.org/archive/X11R7.6/doc/man/man1/Xvfb.1.xhtml) to run browsers headlessly.
-
-The following sections are required in your `.travis.yml` to start `Xvfb`:
-
-```yaml
-dist: bionic
-
-addons:
-  firefox: latest
-  chrome: stable
-
-services:
-  - xvfb
-
-language: node_js
-node_js: 'stable'
-
-before_script:
-  - fluxbox >/dev/null 2>&1 &
-  - sleep 3
-```
-
-You can find more information about Travis and Xvfb in [this article](https://docs.travis-ci.com/user/gui-and-headless-browsers/#Using-xvfb-to-Run-Tests-That-Require-a-GUI).
-
-## Step 4 - Add the `test` script to package.json
+## Step 3 - Add the `test` script to package.json
 
 To test a project, Travis runs test scripts. For Node.js projects, the default test script is `npm test`.
 To tell npm how to run your tests, add the `test` script to the project's package.json file. Use `testcafe` command in the script to run tests in Chrome and Firefox.
 
+> Important! Travis CI uses Ubuntu Server virtual machines that do not have regular graphical environment like Unity, GNOME or KDE installed. Modify the `testcafe` command to run browsers headlessly.
```

Comment:
> Modify the `testcafe` command to run browsers headlessly.

Can we just write "Use the `:headless` prefix to run browsers headlessly."?

Because 'modify' would assume that a user already has a command without `:headless` and now they need to modify this command.

Reply:
Agreed. Although wouldn't this be a postfix rather than a prefix?

Reply:
Oh, sorry, postfix of course 😄

Reply:
Addressed in [f15769d](https://github.com/DevExpress/testcafe/commit/f15769d4ea779dc1d133eee5f75d6114b3049e71)